### PR TITLE
✨ [Amp story] Activate desktop one panel experiment to 1.

### PR DESCRIPTION
**Pull request by processprocess**
Activates `amp-story-desktop-one-panel` to `1`.

### CODE REVIEW

**Review by gmajoulet — COMMENTED** [review]
Can you update `client-side-experiments-config.json` too? That's the one to use if you want the experiment to be instantly flipped

**Comment by gmajoulet** [comment]
cc @danielrozenberg FYI :))

**Comment by danielrozenberg** [comment]
Do you want to immediately set it to 1 in prod/canary config, or gradually ramp it up through the client side experiments?

**Comment by processprocess** [comment]
> Do you want to immediately set it to 1 in prod/canary config, or gradually ramp it up through the client side experiments?

We don't have metrics to observe before deciding to launch, so we're thinking to push it all the way to 1.

**Review by gmajoulet — APPROVED** [review]
The code LGTM, but let's get formal wg-stories PM/UX approval before merging

**Review by danielrozenberg — APPROVED** [review]
Approval for client-side-experiments-config.json

File: build-system/global-configs/canary-config.json
Comment (jridgewell) [inline build-system/global-configs/canary-config.json:25]: Nit: Changes to `canary-config` and `prod-config` shouldn't be necessary anymore.

**Review by jridgewell — APPROVED** [review]

**Review by gmajoulet — APPROVED** [review]
Send it!! 🚢

**Pull request merged**
